config/initializers/bulkrax.rb

```diff
@@ -23,12 +23,16 @@
       'subject' => { from: %w[sm_subject subject], split: true },
       'title' => { from: %w[sm_title title], split: true },
       # Custom property mappings
-      'collection' => { from: %w[bs_iscollection collection], split: true },
```

Comment:
arent these going to break something?

Reply:
TL;DR -- No 

Old Bulkrax had a field mapping called collection, but not anymore. This mapping was for a Mobius-specific field that we ended up handling separately in `HasLocalProcessing`